aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move

```diff
@@ -34,9 +34,21 @@ module aptos_framework::aggregator {
         aggregator.limit
     }
 
+    /// Adds `value` to aggregator. 
+    /// Returns a bool flag indicating whether the operation is successfull. 
+    /// If the output is true, then there is no error in the operation.
+    /// If the output is false, then there is overflow error.
+    public native fun try_add(aggregator: &mut Aggregator, value: u128): bool;
+
     /// Adds `value` to aggregator. Aborts on overflowing the limit.
     public native fun add(aggregator: &mut Aggregator, value: u128);
```

Comment:
We don't want to delete Rust functions because try_ functions use them :) But yes, we should make other two being native, and replace instances of aggregator methods with try_ aggregator methods.

A simple code upgrade is ok because we are not changing signatures, plus aggregators never failed yet so we can add error codes and wrap try_ methods in asserts.

Reply:
@vusirikala can we make `add`/`sub` non-native?

Reply:
@vgao1996 can we remove native functions from the gas schedule table without breaking compatibility?

Reply:
I'd suggest making this change to switch to non-native first then change the gas schedule later for safety

Reply:
Done.